CliFx/Models/CommandOptionSchema.cs

```diff
@@ -38,17 +38,24 @@ public partial class CommandOptionSchema
         /// </summary>
         public string? EnvironmentVariableName { get; }
 
+        /// <summary>
+        /// Optional sample value that will be used in help text for usage of required options. In case this property is not 
+        /// set for a required option option's name will be used as sample value.
+        /// </summary>
+        public string? SampleValue { get; set; }
```

Comment:
I'm thinking, will anyone want to configure this? I feel like as a user of this library, it would be too much effort to set short option name, long option name, description, and then also the sample value.
I vote we just use something generic like `--option <value>` instead in help text. Just to make things simpler.

Reply:
I buy your point, there's always a value in keeping things simple. I removed all `SampleValue` stuff and related test cases and just used `--option <value>` as you suggested